lib/App.js

```diff
@@ -43,7 +53,7 @@ class App {
    *  used as the handler. If a plain object is provided, it will be run through the Route constructor.
    * @returns {App} - app instance for chaining
    */
-  use(route) {
+  then(route) {
```

Comment:
then and catch methods look pmuch exactly the same at this point. Lets delegate to implementation method of `_addMiddleware`